packages/admin/admin-rte/src/core/Controls/LinkControls.tsx

```diff
@@ -52,12 +63,17 @@ export default (p: IControlProps) => {
 };
 
 declare module "@mui/material/styles" {
+    interface ComponentsPropsList {
+        CometAdminRteLinkControls: RteLinkControlsProps;
+    }
+
     interface ComponentNameToClassKey {
         CometAdminRteLinkControls: RteLinkControlsClassKey;
     }
 
     interface Components {
         CometAdminRteLinkControls?: {
+            defaultProps?: ComponentsPropsList["CometAdminRteLinkControls"];
```

Comment:
`defaultProps` should always be `Partial` to prevent having to set all required props in the default-values. 

```suggestion
            defaultProps?: Partial<ComponentsPropsList["CometAdminRteLinkControls"]>;
```